.ci/watchdog/Watchdog.py

```diff
@@ -89,13 +89,14 @@ def __init__(self, jenkins_token, jenkins_server, jenkins_user, git_token, git_o
         # Load GitHub token and log in, retrieve pull requests
         self._git = GitWrapper(git_token, repository=git_org, project=git_project)
         # Create Slack api object
-        self._slack_app = SlackCommunicator(slack_token)
+        proxy = os.environ["http_proxy"] or os.environ["HTTP_PROXY"]
```

Comment:
Why are you detecting the proxy here? Wouldn't it make sense to detect it in the `SlackCommunicator` init function?

Reply:
I didn't like the idea of burying proxy setting somewhere in SlackCommunicator. Thought exposing it as init parameter could make it clearer in case I encounter any problems with connection in the future.